### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by lydell**
- Fix favicon URL
- Remove unused favicon.svg
- Fix typo / sync description
- Remove unnecessary `?inline`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "vite-elm-template",
   "author": "Lindsay Wardell",
-  "description": "A staring template for building Elm apps with Vite",
+  "description": "A default template for building Elm applications using Vite.",
```
Comment (lydell) [inline package.json:4]: Typo: staring -> star<strong>t</strong>ing

But I synced the description with the repo description and the README instead.

File: src/Main.elm [hunk 2]
```diff
@@ -26,6 +26,6 @@ update msg model =
 view : Int -> Html Msg
 view model =
     div []
-        [ img [ src <| VitePluginHelper.asset "/src/assets/logo.png?inline", style "width" "300px" ] []
+        [ img [ src <| VitePluginHelper.asset "/src/assets/logo.png", style "width" "300px" ] []
```
Comment (lydell) [inline src/Main.elm:29]: `?inline` does not seem to do anything. Instead, assets are automatically inlined based on size: https://vitejs.dev/config/build-options.html#build-assetsinlinelimit

**Review by lydell — COMMENTED** [review]
Tested with both `npm run dev` and `npm run build && npm run serve`

**Comment by lindsaykwardell** [comment]
Looks good, thank you!

**Pull request merged**